spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -360,3 +360,212 @@ Another option to start a task using the `task-launcher` would be to create a st
+dataflow:> task create my-composed-task --definition "mytaskapp && timestamp"
+dataflow:> task launch my-composed-task
+```
+In the example above we assume that the applications to be used by our composed
+task have not been registered yet.  So the first two steps we register two task
+applications.  We then create our composed task definition by using the task
+create command.  The composed task DSL in the example above will, when launched,
+execute mytaskapp and then execute the timestamp application.
+
+But before we launch the my-composed-task definition,  we can view what
+Spring Cloud Data Flow generated for us.  This can be done by executing the
+task list command.
+
+```
+dataflow:>task list
+╔══════════════════════════╤═══════════════════════════════════════════════════════════════
+║        Task Name         │                      Task Definition
+╠══════════════════════════╪═══════════════════════════════════════════════════════════════
+║my-composed-task          │mytaskapp && timestamp
+║my-composed-task-mytaskapp│mytaskapp
+║my-composed-task-timestamp│timestamp
+```
+Spring Cloud Data Flow created three task definitions, one for each of the
+applications that comprises our composed task as well as the composed task
+definition.  We also see that each of the generated
+names for the child tasks is comprised of the name of the composed task and
+the name of the application separated by a dash `-`.  i.e. _my-composed-task_ `-`
+_mytaskapp_.
+
+===== Task Application Parameters
+The task applications that comprise the composed task definition can also
+contain parameters.  For example:
+```
+dataflow:> task create my-composed-task --definition "mytaskapp --displayMessage=hello && timestamp --format=YYYY"
+```
+
+==== Launching a Composed Task
+Launching a composed task is done the same way as launching a stand-alone task.
+i.e.
+```
+task launch my-composed-task
+```
+Once the task is launched and assuming all the tasks complete successfully you will
+see three task executions when executing a `task execution list`.  For example:
+```
+dataflow:>task execution list
+╔══════════════════════════╤═══╤════════════════════════════╤════════════════════════════╤═════════╗
+║        Task Name         │ID │         Start Time         │          End Time          │Exit Code║
+╠══════════════════════════╪═══╪════════════════════════════╪════════════════════════════╪═════════╣
+║my-composed-task-timestamp│713│Wed Apr 12 16:43:07 EDT 2017│Wed Apr 12 16:43:07 EDT 2017│0        ║
+║my-composed-task-mytaskapp│712│Wed Apr 12 16:42:57 EDT 2017│Wed Apr 12 16:42:57 EDT 2017│0        ║
+║my-composed-task          │711│Wed Apr 12 16:42:55 EDT 2017│Wed Apr 12 16:43:15 EDT 2017│0        ║
+╚══════════════════════════╧═══╧════════════════════════════╧════════════════════════════╧═════════╝
+```
+In the example above we see that my-compose-task launched and it also launched
+the other tasks in sequential order.
```

Comment:
Perhaps also end the statement with: 

"... and all of them executed successfully with "Exit Code" as `0`"